README.md

```diff
@@ -3,15 +3,16 @@
 The Download Directory step generates a number of files that can be downloaded from Reactome's <a href="https://reactome.org/download-data">download page</a>. It also generates an archive file during the <b>CreateReleaseTarball</b> step that acts as a snapshot of the <a href="https://github.com/reactome/Release">Release repository</a> and some services on the release server. This module has been rewritten from Perl to Java. 
```

Comment:
This is meant to be a comment regarding the entire README: We probably want a blurb in here about 'HumanPathwaysWithDiagrams' file that is generated for curators. Maybe we can have a section denoted 'Files generated for internal use' or something. I can run through READMEs as well in the future and ensure that we're up to date

Reply:
https://github.com/reactome/release-download-directory/pull/2/commits/e2af94f2f4035e0be1b84f6f2976ad60947d1ab9